### PR TITLE
[4.0]housekeeping

### PR DESCRIPTION
**Pull request by brianteeman**
fixes various typos and grammar mainly in code blocks and comments

### CODE REVIEW

File: tests/Unit/Libraries/Cms/Feed/FeedParserTest.php [hunk 1]
```diff
@@ -388,7 +388,7 @@ public function testMoveToClosingElementWithSelfClosingTag()
 	 */
 	protected function getXmlReader($xml, $moveTo): XMLReader
 	{
-		// Its hard to mock the xml reader stream, so we use the real object here (but set xml directly)
+		// it's hard to mock the xml reader stream, so we use the real object here (but set xml directly)
```
Comment (Quy) [inline tests/Unit/Libraries/Cms/Feed/FeedParserTest.php]: Capitalize?

Comment (brianteeman) [reply tests/Unit/Libraries/Cms/Feed/FeedParserTest.php]: thanks

Comment (Quy) [reply tests/Unit/Libraries/Cms/Feed/FeedParserTest.php]: There are more instances of this.

Comment (brianteeman) [reply tests/Unit/Libraries/Cms/Feed/FeedParserTest.php]: sorry - over aggressive replace - fixed now

File: administrator/components/com_menus/Controller/MenusController.php [hunk 2]
```diff
@@ -227,7 +227,7 @@ public function resync()
 					$componentId = -1;
 				}
 
-				// Check for mis-matched component id's in the menu link.
+				// Check for mis-matched component ids in the menu link.
```
Comment (Quy) [inline administrator/components/com_menus/Controller/MenusController.php:230]: Here we have it as `IDs`. There are other instances in this PR. https://github.com/joomla/joomla-cms/pull/27264/files#diff-160b8c38ac42492a5bfc26e902961af1R266

Comment (brianteeman) [reply administrator/components/com_menus/Controller/MenusController.php:230]: it is mixed all over the place - I was just fixing the grammar

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ba8487c717f20dbc1065204bc5c45ed0a76f8b5c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27264">issues.joomla.org/tracker/joomla-cms/27264</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on ba8487c717f20dbc1065204bc5c45ed0a76f8b5c<br /><br />I'm not a native English speaker, but I think my knowledge in English is good enough to confirm that all these changes are correct.

If someone thinks this is not sufficient for a good test, let me know and I'll change back my test result to "Not tested".<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27264">issues.joomla.org/tracker/joomla-cms/27264</a>.</sub>

**Comment by SharkyKZ** [comment]
RTC.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27264">issues.joomla.org/tracker/joomla-cms/27264</a>.</sub>

**Comment by brianteeman** [comment]
thanks for testing

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**